user-guide/Cloud_Platform/Catalog/Tutorials/Tutorial_Register_Catalog_Item.md

```diff
@@ -0,0 +1,166 @@
+---
+uid: Tutorial_Register_Catalog_Item
+---
+
+# Registering a new catalog item
+
+This tutorial demonstrates how to add a new catalog item using the [Catalog API](xref:Catalog_Registration).  
+We will be registering our own version of the following example [connector](https://github.com/SkylineCommunications/SLC-C-Example_Rates-Custom).  
+
+For ease of use we will be using [Postman](https://www.postman.com/) to execute the register API calls. Feel free to integrate them afterwards in your workflow, e.g by using them in a github action.
```

Comment:
We could link to Bruno, this seems to be an open source competitor
Can be installed an ran without an account, even has a portable option

We can then also provide a collection that they can import directly.
Here we can prefill everything and/or put placeholders (e.g. their key)

https://www.usebruno.com/

Reply:
will try it out

Reply:
looked into bruno and my first impression is that postman looks more professional atm and is well known , we can export the queries of it as well. I don't see an advantage of bruno over postman.  Not convinced if we should share it though, its something extra to maintain and tutorial will change to replace field x to xxx iso logical steps. They dont have to do much as is. The tutorial can be done using postman but is not required, they got all steps to do it with anything else.

Reply:
Ok, can keep the link to Postman, it's indeed a lot more known than Bruno atm.

Biggest benefit I saw was that everything is saved locally and it doesn't require an account